lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2059,13 +2059,25 @@ class HSADevice final : public KalmarDevice
         std::string str(fun);
```

Comment:
@weixingzhang it appears `HSADevice::CreateKernel()` doesn't really honor the revised interface in `kalmar_runtime.h`.